pylib/Tools/CNC/IPMITool.py

```diff
@@ -300,7 +300,7 @@ def execute(self, log, keyvals, testDef):
         self.threads = []
         # release the queue
         self.lock.release()
-        if len(targets) < self.options['numthreads']:
```

Comment:
correct way is using `cmds['numthreads']`. -- `cmds` should have the final value for numthreads